### ISSUE TITLE
Not applying

### ISSUE THREAD

**Issue by alaris-ferrera** [EliverLara/firefox-nordic-theme#1]
Hi!
So I tried to install this theme but the script isn't installing it for me. I'm on macOS Monterey, with the latest version of firefox. Do I need to be on a specific version or something for this to work? When I run the script it says it worked, but nothing applies, and firefox remains the same.

**Comment by Eccentrici** [comment]
Same issue, I'm on Arch Linux doing this. I get no error if I run from the terminal, but the theme isn't applied. The best option would be doing a manual install.

**Comment by EliverLara** [comment]
Hi, I think it was installed in the wrong profile folder please try to run the install script using the `-p` option to define your actual profile name( you can get it by going to `about:support` in firefox)

**Comment by mgor** [comment]
I had the same problem, realized that my actual profile was `.default-release`, and the theme had been installed for a profile actually named `.default`.

**Comment by alaris-ferrera** [comment]
> Hi, I think it was installed in the wrong profile folder please try to run the install script using the `-p` option to define your actual profile name( you can get it by going to `about:support` in firefox)

Ok I will try this after I’m done with school. Thanks alot.

**Comment by derusapce** [comment]
Same, doesn't work with clean, freshly installed firefox on arch-based. 

Manual installation did not help. Specifying a folder did not help either. Tried redoing this method - https://github.com/Aris-t2/CustomJSforFx for this theme, except for the fact that search bar went up and little broken, nothing happened :(

**Comment by 0xsapphir3** [comment]
Yeah same here, I am on Arch Linux too. Tried to install this awesome theme but it looks like the styles (colors) aren't appearing properly

**Comment by mosaleh52** [comment]
same her i have tried both folders
